src/plugins/intel_cpu/src/internal_properties.hpp

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/runtime/intel_cpu/properties.hpp"
+#include "openvino/runtime/properties.hpp"
+
+namespace ov {
+namespace intel_cpu {
+
+/**
+ * @brief Defines how many records can be stored in the CPU runtime parameters cache per CPU runtime parameter type per
+ * stream.
+ * @ingroup ov_dev_api_plugin_api
```

Comment:
it's not a plugin API, nut rather CPU internal properties

Plugin API is API shared across all the plugins and located in https://github.com/openvinotoolkit/openvino/tree/master/src/inference/dev_api